provider/source/src/tests/make_testdata.rs

```diff
@@ -94,7 +94,10 @@ struct ZeroCopyCheckExporter {
 // Such types contain some data that was allocated during deserializations
 //
 // Every entry in this list is a bug that needs to be addressed before stabilization.
-const EXPECTED_VIOLATIONS: &[DataMarkerInfo] = &[];
+const EXPECTED_VIOLATIONS: &[DataMarkerInfo] = &[
+    icu::experimental::dimension::provider::currency::CurrencyEssentialsV1Marker::INFO,
+    icu::experimental::dimension::provider::percent::PercentEssentialsV1Marker::INFO,
+];
```

Comment:
Hm? Why is there a violation?

Reply:
Because I haven't fixed the borrowing yet.